src/torch/README.md

```diff
@@ -0,0 +1,131 @@
+# SIRF-PyTorch Wrapper
+This wrapper provides a bridge between the [SIRF](https://github.com/SyneRBI/SIRF) (Synergistic Image Reconstruction Framework) library and PyTorch, enabling the use of SIRF's image reconstruction operators and objective functions within PyTorch's automatic differentiation (autodiff) framework.
+
```

Comment:
Move statement below on which `nn.Module`s are provided here. That and the stuff on dimensions might be the only thing most people want to know.

Reply:
moved